### DISCUSSION TITLE
Log file for YoURLs is over 1 gb can I empty it?

### DISCUSSION THREAD

**Discussion by phillycodehound** [YOURLS/YOURLS#3797] [Q&A] [answered]
Not very well versed in MySQL. The log-file is more than 1 gb. Do I need all of that data? Or can I empty it?

**Comment by dgw** [comment, marked as answer]
Databases have different types of "log" or "journal" files, some of which you should not delete. If  it's literally a log file, plain text you can read that just shows events happening in the database, it is probably safe to remove.

Telling us the path of the file you think is too large would help answer the question.

**Reply by phillycodehound** [reply]
that worked!
